thirtyinch/src/main/java/net/grandcentrix/thirtyinch/distinctuntilchanged/DistinctUntilChangedInvocationHandler.java

```diff
@@ -28,15 +29,15 @@
 
     private static final String TAG = DistinctUntilChangedInvocationHandler.class.getSimpleName();
 
-    private HashMap<String, Integer> mLatestMethodCalls = new HashMap<>();
+    private HashMap<String, WeakReference<Object[]>> mLatestMethodCalls = new HashMap<>();
 
     private final V mView;
 
-    public DistinctUntilChangedInvocationHandler(V view) {
+    DistinctUntilChangedInvocationHandler(V view) {
```

Comment:
Don't change the visibility. `DistinctUntilChangedInvocationHandler` is part of the public API and should remain there